libs/designer/src/lib/core/state/workflow/workflowSelectors.ts

```diff
@@ -118,6 +118,20 @@ export const getWorkflowNodeFromGraphState = (state: WorkflowState, actionId: st
   return traverseGraph(graph);
 };
 
+export const getWorkflowNodeIndexFromGraphLevel = (
+  workflowGraphNodes: Record<string, WorkflowNode>,
+  runAfters: string[],
+  actionId: string
+) => {
+  const sortedRunAfters = runAfters.slice(0).sort(function compareFn(id1, id2) {
```

Comment:
Nit: You can make this an arrow function
```suggestion
  const sortedRunAfters = runAfters.slice(0).sort((id1, id2) => {
```